### ISSUE TITLE
C# language version is not respected (assignment to readonly property in ctor)

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#4696]
To reproduce:
1.     Create C# project in VS2015
2.     Set language version to C# 5.0
3.     Write Code:
```cs
    public class Class1
    {
        public Class1()
        {
            Prop = "Test";
        }
 
        public string Prop { get; }
    }
```
Expected behaviour: Building results in a compile error in VS2015, since it is not valid C# 5.0
Actual behaviour: Compiles fine with VS2015 but there is a compile error in VS2013

Migrated from TFS/DevDiv 1209958

**Comment by AlekseyTs** [comment]
@gafter Please verify.
